src/TBuffer.cpp

```diff
@@ -2417,7 +2439,7 @@ void TBuffer::translateToPlainText(std::string& incoming, const bool isFromServe
                         break;
                     case 2:
                         isTwoTCharsNeeded = true;
-                    // Fall-through
+                        [[clang::fallthrough]];
```

Comment:
Ah! But the title of the post is `Using C++17 Attributes Today` and that was a macro we could use in place before we adopt C++17 minimum requirement :)